.github/workflows/main.yml

```diff
@@ -711,13 +709,12 @@ jobs:
         set CMAKE_GENERATOR=Ninja
         set CC=clang
         set CXX=clang++
-        set NO_TCMALLOC=On
         set PREFIX=%GITHUB_WORKSPACE%\install
         set CPU_CORES=%NUMBER_OF_PROCESSORS%
 
         set MAKE_DIR=C:\make\bin
         set PATH=%pythonLocation%;%JAVA_HOME%\bin;%MAKE_DIR%;%PATH%
-        set ADDITIONAL_CMAKE_OPTIONS=-DPython3_ROOT_DIR=%pythonLocation%
+        set ADDITIONAL_CMAKE_OPTIONS="-DPython3_ROOT_DIR=%pythonLocation% -DSURELOG_WITH_TCMALLOC=Off"
```

Comment:
Is this the correct way to quote environment variables with spaces @hs-apotell ?

Reply:
Ok, so removing the quotes helped.